actionpack/lib/action_controller/railtie.rb

```diff
@@ -24,7 +24,7 @@ class Railtie < Rails::Railtie #:nodoc:
     initializer "action_controller.parameters_config" do |app|
       options = app.config.action_controller
 
-      ActiveSupport.on_load(:action_controller) do
+      ActiveSupport.on_load(:action_controller_base) do
```

Comment:
What if we stop deleting / mutating the values in the callback? Then we wouldn't need to introduce the complexity of `only_once`.